src/include/uservarmanager.h

```diff
@@ -14,6 +14,15 @@
     #include "globals.h"
 #endif
```

Comment:
Changes to this file means that you have to raise the version number in cbplugin.h. Do it as a last step before push.